admin/src/services/api.js

```diff
@@ -10,6 +10,7 @@ let fetch = window.fetch;
 class api {
   constructor() {
     this.token = "";
+    this.personnalizedHeader = { "user-timezone": new Date().getTimezoneOffset() };
```

Comment:
"customHeader" plutôt non ?

Reply:
La convention est de préfixer les custom headers par `x-` donc `x-user-timezone` (ou plus simplement `x-user-tz`) et je mettrais tout simplement `this.headers`

```suggestion
    this.headers = { "x-user-timezone": new Date().getTimezoneOffset() };
```